metricbeat/module/jolokia/jmx/data_test.go

```diff
@@ -222,6 +222,42 @@ func TestEventGroupingMapperGetRequest(t *testing.T) {
 	assert.ElementsMatch(t, expected, events)
 }
 
+// TestEventGroupingMapperGetRequestUptime tests responses which are returned
+// from a Jolokia GET request and only has one uptime runtime value.
+func TestEventGroupingMapperGetRequestUptime(t *testing.T) {
+	absPath, err := filepath.Abs("./_meta/test")
+
+	assert.NotNil(t, absPath)
+	assert.Nil(t, err)
+
+	jolokiaResponse, err := ioutil.ReadFile(absPath + "/jolokia_get_response_uptime.json")
+
+	assert.Nil(t, err)
+
+	var mapping = AttributeMapping{
+		attributeMappingKey{"java.lang:type=Runtime", "Uptime"}: Attribute{
+			Field: "runtime.uptime", Event: "runtime"},
+	}
+
+	// Construct a new GET response event mapper
+	eventMapper := NewJolokiaHTTPRequestFetcher("GET")
+
+	// Map response to Metricbeat events
+	events, err := eventMapper.EventMapping(jolokiaResponse, mapping)
+
+	assert.Nil(t, err)
+
+	expected := []common.MapStr{
+		{
+			"runtime": common.MapStr{
+				"uptime": float64(88622),
+			},
+		},
+	}
+
+	assert.ElementsMatch(t, expected, events)
+}
+
```

Comment:
Thanks for adding this test!

Could you please add also this case to the integration tests?
```
diff --git a/metricbeat/module/jolokia/jmx/jmx_integration_test.go b/metricbeat/module/jolokia/jmx/jmx_integration_test.go
index e8cff3b62..1677395e4 100644
--- a/metricbeat/module/jolokia/jmx/jmx_integration_test.go
+++ b/metricbeat/module/jolokia/jmx/jmx_integration_test.go
@@ -184,6 +184,15 @@ func getConfigs() []map[string]interface{} {
                                                },
                                        },
                                },
+                               {
+                                       "mbean": "java.lang:type=Runtime",
+                                       "attributes": []map[string]string{
+                                               {
+                                                       "attr":  "Uptime",
+                                                       "field": "uptime",
+                                               },
+                                       },
+                               },
                        },
                },
        }
```

Reply:
Will do! This will be helpful!